src/lib/shell/streamer_esp32.cpp

```diff
@@ -44,6 +44,74 @@ static int chip_command_handler(int argc, char ** argv)
     return static_cast<int>(err.AsInteger());
 }
 
+void get_command_completion(const char * buf, linenoiseCompletions * lc)
+{
+    size_t len = strlen(buf);
+
+    // If there is a trailing space, remove it for completion lookup
+    bool trailing_space = false;
+    if (buf[len - 1] == ' ')
+    {
+        trailing_space = true;
+        memset((char *) &buf[len - 1], 0, 1);
```

Comment:
this is a very complex way of saying `buf[len-1] = 0`. Please replace with simple assignment